Talend/ui
# chore: upgrade to webpack 5

**What is the problem this PR is trying to solve?**

utdated tooling packages due to webpack 4

security issue:
webpack4 => watchpack => ... => glob-parent

**What is the chosen solution to this problem?**

* Upgrade to webpack 5

QA validations tasks:
* yarn list --pattern webpack is as flat as possible ✅ 
* yarn list --pattern loader is as flat as possible ✅
* yarn postinstall ✅ (icons, tokens, theme are OK)
* yarn run pre-release ✅ (check dependencies.json are generated along report and map files)

storybook still use webpack 4. WIP on it to analyze the impact to use builder wepback5 on it.


**Please check if the PR fulfills these requirements**

- [ ] The PR have used `yarn changeset` to a request a release from the CI if wanted.
- [ ] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR



**[ ] This PR introduces a breaking change**
